tests/glsl-intrinsic/shader-subgroup/shader-subgroup-arithmetic.slang

```diff
@@ -0,0 +1,219 @@
+//TEST:SIMPLE(filecheck=CHECK_GLSL):  -allow-glsl -stage compute -entry computeMain -target glsl -DTARGET_GLSL
+//TEST:SIMPLE(filecheck=CHECK_SPV):  -allow-glsl -stage compute -entry computeMain -target spirv -emit-spirv-directly -DTARGET_SPIRV
```

Comment:
I don't think -D here will work as expected. You need to wrap it with -xslang to pass it through to the slang compiler. Here you are just specifying command line args to slang-test.

should be:
```
-xslang -DTARGET_SPIRV
```